prow/config/config.yaml

```diff
@@ -490,3 +491,78 @@ presubmits:
         resources:
           requests:
             memory: "500Mi"
+  
+  metal3-io/hardware-classification-controller:
+  - name: gofmt
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/gofmt.sh
```

Comment:
you do not have those scripts in place in your repo. You have to add them before being able to use them

Reply:
I think the plan is to add the test jobs, then use those jobs to test the scripts as they are added to the repo.

Reply:
Yes. +1